app/src/org/commcare/views/media/MediaLayout.java

```diff
@@ -144,12 +146,12 @@ private void showMediaAboveText() {
         alignMediaAtTop(qrView);
         alignMediaAtTop(imageView);
         alignMediaAtTop(resizingImageView);
+        alignMediaAtTop(missingMediaBackground);
         alignMediaAtTop(downloadIcon);
         alignMediaAtTop(progressBar);
 
-        // Next align the text, audiobutton and videoButton below mediaView.
+        // Next align the text, audiobutton below mediaView.
         alignTextContainerBelowMediaView(audioButton);
-        alignTextContainerBelowMediaView(videoButton);
```

Comment:
confused why did we remove `videoButton` from here

Reply:
`videoButton` has a constraint which places it below `audioButton`. So we only need to align `audioButton` top constraint while shifting it below media view.